docs/oss_rules_of_play_rating.md

```diff
@@ -0,0 +1,65 @@
+# Rules of play ratings for open source projects
+
+This section describes a rules of play rating for open source projects.
+The rating is implemented in the [OssRulesOfPlayRating](https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/oss/phosphor/fosstars/model/rating/oss/OssRulesOfPlayRating.java) class.
+
+The rating assesses how well projects care about open source best practices and community health aspects. 
+The rating may be used to estimate how well-administered an open source project is and how it adheres to recommendations how an open source project should be set up.
```

Comment:
These 2 points should come in the beginning of this section I think.

Reply:
Done, thanks!